### PR TITLE
Update cast icon

### PR DESCRIPTION
**Pull request by hghazzi**
Updated css of cast icon to look better on both android and desktop

JW7-3721

### CODE REVIEW

File: src/css/imports/controlbar.less [hunk 1]
```diff
@@ -94,6 +94,9 @@
   display: none;
   margin: 0;
   padding: 0;
+  width: 32px;
```
Comment (robwalch) [inline src/css/imports/controlbar.less]: Can we use `em`s? 32px when not zoomed (16px fontsize) is 2em.

Comment (robwalch) [inline src/css/imports/controlbar.less]: Use Command + / Command - (Menu > View > Zoom In/Out) to compare the icon to it's neighbors.

**Comment by hghazzi** [comment]
@robwalch updated

**Comment by robwalch** [comment]
Thanks!

**Pull request merged**